web/static/css/app.css

```diff
@@ -344,6 +344,12 @@ footer p a {
   background-image: url("../images/whatsapp_hover.svg");
 }
 
+
+.blurred_background {
+  background-image: url("../images/blurred_gradient.png");
+  background-repeat: no-repeat;
```

Comment:
Alternative in tachyons is `bg-center` which centers the background but also sets no-repeat.